client/doc.go

```diff
@@ -29,7 +29,7 @@ The simplest way to use the client is through the Call method. Call
 synchronously invokes the method and returns the reply and an
 error. The example below shows a get and a put.
 
-  kv := client.NewKV(client.NewHTTPSender("localhost:8080", tlsConfig))
+  kv := client.NewKV(client.NewHTTPSender("localhost:8080", tlsConfig), clock)
```

Comment:
Probably better for example purposes to set clock to "nil" instead, which causes it to default to using time.Now().UnixNano()